extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -240,6 +246,10 @@ interface InitialState {
   accountBalances: AccountBalancesInterface;
   destinationBalances: AccountBalancesInterface;
   assetIcons: AssetIcons;
+  assetSelect: {
+    type: AssetSelectType;
+    source: boolean;
```

Comment:
can we call this `isSource` (or something similar) since it's a bool?

Reply:
ooh ya  good call https://github.com/stellar/freighter/pull/604/commits/cbd168ded8a0de121d0690ff88f178531c09eb09